added fix to docker image (#1176)
demisto/demisto-sdk: CHANGELOG.md

#Changelog

> * Fixed an issue where **update-release-notes** on newly created integration added release notes with '- Updated the Docker image' note.

Reviewer: ```suggestion
* Fixed an issue where **update-release-notes** on newly created integration added release notes for docker update.
```

# Changelog
* Fixed an issue where **lint** failed where *.Dockerfile* exists prior running the lint command.
* Added FeedHelloWorld template option for *--template* flag in **demisto-sdk init** command.

> * Fixed an issue where **update-release-notes** on newly created integration added release notes for docker update.

Reviewer: ```suggestion
* Fixed an issue where running **update-release-notes** on a pack with newly created integration, had also added a docker image entry in the release notes.
```
